### ISSUE TITLE
Use of obsolete TeX function

### ISSUE THREAD

**Issue by olexandr-konovalov** [gap-packages/hecke#2]
Loading in GAP with `-O` option causes a warning:
```
Syntax warning: Unbound global variable in /circa/scratch/gap-jenkins/workspac\
e/GAP-pkg-update-stable-quicktest/GAPCOPTS/64build/GAPGMP/gmp/GAPTARGET/obsole\
tes/label/kovacs/GAP-pkg-update-stable-snapshot/pkg/hecke/gap/output.gi:240
      PrintFn:=function(x) Append(str, TeX(x)); end;
                                          ^
```
because `TeX` is obsolete. Need to understand what is the purpose of its usage there.

**Comment by fingolfin** [comment]
Interestingly, this uses `TeX`, which in turn uses `TeXObj`, for which there are no methods. So this code just doesn't work right now.

The comments there, however, talks about "LaTeX output"... and for `LaTeX` resp. `LaTeXObj` there are some non-trivial methods installed. Case in points:
```
gap> LaTeX([[1,2]]);
"\\left(\\begin{array}{rr}%\n1&2\\\\%\n\\end{array}\\right)%\n"
gap> TeX([[1,2]]);
Error, no method found! For debugging hints type ?Recovery from NoMethodFound
```

**Comment by olexandr-konovalov** [comment]
The code from hecke/gap/output.gi:240 

https://github.com/gap-packages/hecke/blob/3049f50db750a5c2e53689da8f2d64d588fc7984/gap/output.gi#L210-L220 

says that it is for printing a decomposition matrix, and it can print it in two ways. Interesting whether the 2nd way can be done by [`LaTeXStringDecompositionMatrix`](https://www.gap-system.org/Manuals/doc/ref/chap71_mj.html#X83EC921380AF9B3B).

**Comment by olexandr-konovalov** [comment]
Also, the argument `d` is not a record as comments says - it has `IsDecompositionMatrix`. Perhaps the comment is a relic from GAP 3.

**Comment by olexandr-konovalov** [comment]
Maybe removing this code, together with merged #6 can form a new release of this package...
